deploy/eck-agent/values.yaml

```diff
@@ -45,7 +45,7 @@ spec:
   # Reference to ECK-managed Elasticsearch instance.
   #
   elasticsearchRefs:
-  - name: elasticsearch
+  - name: eck-elasticsearch
```

Comment:
These are simply the default values, and are just placeholders for users to provide their own names, as the names of these follow these rules from the values files: (they, by default, are prefixed with the `Release` name that the user provides, so there's no way to predict it)

```
# Overridable names of the Elasticsearch resource.
# By default, this is the Release name set for the chart,
# followed by 'eck-elasticsearch'.
#
# nameOverride will override the name of the Chart with the name set here,
# so nameOverride: quickstart, would convert to '{{ Release.name }}-quickstart'
#
# nameOverride: "quickstart"
#
# fullnameOverride will override both the release name, and the chart name,
# and will name the Elasticsearch resource exactly as specified.
#
# fullnameOverride: "quickstart"
```

Since we are also incrementing the semver already, it may make sense to indicate that these are breaking changes, if any users are using this default behavior.  I'll do so.